symfony/src/Controller/DeceasedReportsController.php

```diff
@@ -109,6 +111,30 @@ public function deceasedReportNew(Request $request, ParticipantSummaryService $p
         ]);
     }
 
+    /**
+     * @Route("/stats", name="deceased_report_stats")
+     */
+    public function getStats(SessionInterface $session, DeceasedReportsService $deceasedReportsService)
+    {
+        $organizationId = $session->get('siteOrganizationId');
+        $cache = new DatastoreAdapter(100);
+        $pendingReportCountCache = $cache->getItem('deceased_reports.pending.count');
+        if ($cacheHit = $pendingReportCountCache->isHit()) {
+            $pendingReportCount = (int) $pendingReportCountCache->get();
+        } else {
+            $pendingReportCountCache->expiresAfter(60);
```

Comment:
Can increase this to a higher number, but since reports can come from platforms other than HealthPro, we could wind up with an inaccurate count.

Reply:
I'm thinking at least 5 minutes would be better. Otherwise, that's a lot of requests off to the RDR every time the home page is loaded.

We could consider some more complex caching logic, too. Maybe set the TTL longer if the count is 0 and shorter if it is non-0. And we could clear the organization's cache value whenever a deceased report is submitted or reviewed, since we know for sure it will change at that point.

Reply:
Will up it to five minutes. We can clear _our_ version of the cache, but would we have overlap with PMT users?

Reply:
We can take this back to Caroline, but IMO, 5 minutes is an acceptable delay. And if we do some optimization based on the stuff we do know, even better.

Reply:
Addressed in 3d7a541. Can bring it up in tomorrow's meeting to see if it's a major concern.